framework/db/ActiveRecord.php

```diff
@@ -199,34 +200,56 @@ protected static function findByCondition($condition)
      * This method will ensure that an array condition only filters on existing table columns.
      *
      * @param array $condition condition to filter.
+     * @param array $aliases
      * @return array filtered condition.
      * @throws InvalidArgumentException in case array contains unsafe values.
+     * @throws InvalidConfigException
      * @since 2.0.15
      * @internal
      */
-    protected static function filterCondition(array $condition)
+    protected static function filterCondition(array $condition, array $aliases)
```

Comment:
Changing signature this way would break backwards compatibility. How about `$aliases = []`?

Reply:
of course